packages/react-ui-components/src/components/SchemaForm/Widgets/AdvancedDateSelectDropdownWidget/AdvancedDateSelectDropdownWidget.tsx

```diff
@@ -0,0 +1,57 @@
+import { ICalendarAdvanceSelectedTime, ITimeChange } from '@kelvininc/ui-components';
+import { WidgetProps } from '@rjsf/utils';
+import { get } from 'lodash';
+import React, { useMemo, useState } from 'react';
+import { KvAdvancedDateSelectDropdown, KvFormLabel } from '../../../stencil-generated';
+import { DEFAULT_DROPDOWN_POSITION_OPTIONS } from './config';
+import { getDefaultTimezone } from './utils';
+
+const AdvancedDateSelectDropdownWidget = ({ schema, label, id, disabled, required, onChange, uiSchema }: WidgetProps) => {
+	const {
+		inputConfig,
+		selectedTime,
+		absoluteTimeConfig,
+		relativeTimeConfig,
+		selectedTimezone = getDefaultTimezone(),
+		dateMask,
+		dropdownPositionOptions = DEFAULT_DROPDOWN_POSITION_OPTIONS
+	} = schema;
```

Comment:
Can you have this props defined inside the schema? 🤔 Please test it.

Reply:
I actually changed it to uiSchema.